### PR TITLE
Added updateoperator function to streamtool

### PR DESCRIPTION
**Pull request by Karsten12**
- All test cases pass

Minor note: 
the test_parallelRegionWidth checks that the passed in parallel region width (the --parallelRegionWidth arg) overrides the one in the JCO, and I've confirmed it works (via printing out the JCO after the override), but this print statement has been removed, since its used for debugging only. Thus this test is essentially useless.
When the 1Q20 fix is implemented, and the proper JSON results of the updateoperator function is returned,  I will have to update this test to check for the correct parallel region width

### CODE REVIEW

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py [hunk 1]
```diff
@@ -2780,6 +2791,24 @@ def _cancel_job(self, job, force):
 
         return False
 
+    def _update_operators(self, job, job_config):
+        self.rest_client._block_ssl_warn()
+
+        job_options = job_config.as_overlays() if job_config else {}
```
Comment (ddebrunner) [inline com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py]: Does an empty job_options make sense?

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py [hunk 2]
```diff
@@ -2780,6 +2791,24 @@ def _cancel_job(self, job, force):
 
         return False
 
+    def _update_operators(self, job, job_config):
+        self.rest_client._block_ssl_warn()
```
Comment (ddebrunner) [inline com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py]: Should be removed before final merge.

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py [hunk 3]
```diff
@@ -2780,6 +2791,24 @@ def _cancel_job(self, job, force):
...
+
+        job_options = job_config.as_overlays() if job_config else {}
+
+        # Update the job operators using the job id
+        update_url = job.instance + '/jobs/' + job.id
+        res = self.rest_client.session.patch(update_url,
+                headers = {'Accept' : 'application/json'},
+                json = {'jobConfigurationOverlay':job_options, 'preview':False},
```
Comment (ddebrunner) [inline com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py]: `preview` should be passed in, defaulting to `False`.

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py (hunk 2 above)
Comment (ddebrunner) [reply com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py]: This call is needed, sorry I thought this was always disabling SSL.

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py (hunk 1 above)
Comment (Karsten12) [reply com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py]: Removed in 6b53625

**Pull request merged**